src/llmcompressor/modeling/deepseek_v3.py

```diff
@@ -46,7 +45,7 @@ def forward(self, hidden_states):
         return hidden_states
 
 
-def replace(module: DeepseekV3MoE) -> DeepseekV3MoECalibrate:
-    return DeepseekV3MoECalibrate(
+def replace(module: "DeepseekV3MoE") -> DeepseekV3MoE:
```

Comment:
Aren't you replacing module of type `transformers.models.deepseek_v3.modeling_deepseek_v3.DeepseekV3MoE` with one of type `llmcompressor.modeling.deepseek_v3.DeepseekV3MoE` here? Why the change in signature?

Reply:
Oh I see what you're saying